include/RAJA/policy/openmp/atomic.hpp

```diff
@@ -68,17 +68,37 @@ template <typename T>
 RAJA_HOST_DEVICE
 RAJA_INLINE T atomicMin(omp_atomic, T volatile *acc, T value)
 {
+#if _OPENMP >= 202011
+  T ret;
```

Comment:
I think I'd also like to change "ret" to "old" to make it clearer that we are returning the old value. I've left that change out for now to avoid obscuring the changes of substance. If you agree that it would be clearer, then I can change that in this merge request or a follow on. Otherwise, I won't worry about making the change.

Reply:
I went ahead and made the change.